docs/docs/products/pablo-overview.md

```diff
@@ -6,7 +6,7 @@ provide a complete suite of features that together form a protocol that is great
 
 Pablo offers multiple AMM-types, enhanced customizability for LPs and traders, liquidity-bootstrapping pools, 
 cross-chain assets through our IBC and EVM bridges, and single sided staking.
-Upon launch, Balancer pools based on [Balancers weighted math] will be available,
+Upon launch, Balancer pools based on [BalancerFi weighted math] will be available,
```

Comment:
didn't we say xy=k pools will be available on launch and not balancer pools.

Reply:
Will replace with the final suggestion from @vimukthi-git previously.